### ISSUE TITLE
test_dna and test_pydock failed.

### ISSUE THREAD

**Issue by alchemistcai** [lightdock/lightdock#54]
**Describe the bug**

I tried to add brackets after residues' is_* methods in the `lightdock.structure.complex.Complex.get_atoms` function's defination as they are not properties,but `lightdock.test.bin.test_dna.Test*` and `lightdock.test.bin.test_pydock.Test*` tests failed.

These tests didn't pass atom sanity checks in the `lightdock.structure.nm.calculate_nmodes` function because of the different atom indice.

**To Reproduce**

I tried to change the codes as below:

```py
# in lightdock.structure.complex.Complex.get_atoms
    def get_atoms(
        self, protein: bool = True, nucleic: bool = True, dummy: bool = False
    ):
        """Selects atoms on structure depending on their nature"""
        atoms: "list[Atom]" = []
        for residue in self.residues:

#  original 
            if residue.is_standard and protein:
                atoms.extend(residue.atoms)
            elif residue.is_nucleic and nucleic:
                atoms.extend(residue.atoms)
            elif residue.is_dummy and dummy:
                atoms.extend(residue.atoms)
        return atoms

# new 
            if residue.is_standard() and protein:
                atoms.extend(residue.atoms)
            elif residue.is_nucleic() and nucleic:
                atoms.extend(residue.atoms)
            elif residue.is_dummy() and dummy:
                atoms.extend(residue.atoms)
        return atoms
```

Nose tests outputs are as below:

```log
lightdock.test.bin.test_dna.TestRegressionDNARestraints.test_lightdock_1diz_30_steps_50_glowworms ... usage: lightdock3 [-h] [-f configuration_file] [-s SCORING_FUNCTION]
                  [-sg GSO_SEED] [-t TRANSLATION_STEP] [-r ROTATION_STEP] [-V]
                  [-c CORES] [--profile] [-mpi] [-ns NMODES_STEP] [-min]
                  [--listscoring] [-l SWARM_LIST [SWARM_LIST ...]]
                  setup_file steps
lightdock3: error: argument setup_file: The file does not exist
ERROR
lightdock.test.bin.test_dna.TestRegressionDNAShort.test_lightdock_1diz_10_steps_20_glowworms ... usage: lightdock3 [-h] [-f configuration_file] [-s SCORING_FUNCTION]
                  [-sg GSO_SEED] [-t TRANSLATION_STEP] [-r ROTATION_STEP] [-V]
                  [-c CORES] [--profile] [-mpi] [-ns NMODES_STEP] [-min]
                  [--listscoring] [-l SWARM_LIST [SWARM_LIST ...]]
                  setup_file steps
lightdock3: error: argument setup_file: The file does not exist
ERROR

lightdock.test.bin.test_pydock.TestRegressionPyDockRestraints.test_lightdock_2uuy_20_steps_25_glowworms_rst ... usage: lightdock3 [-h] [-f configuration_file] [-s SCORING_FUNCTION]
                  [-sg GSO_SEED] [-t TRANSLATION_STEP] [-r ROTATION_STEP] [-V]
                  [-c CORES] [--profile] [-mpi] [-ns NMODES_STEP] [-min]
                  [--listscoring] [-l SWARM_LIST [SWARM_LIST ...]]
                  setup_file steps
lightdock3: error: argument setup_file: The file does not exist
ERROR

======================================================================
ERROR: lightdock.test.bin.test_dna.TestRegressionDNARestraints.test_lightdock_1diz_30_steps_50_glowworms
----------------------------------------------------------------------

Traceback (most recent call last):
  File "/home/regen/ana/envs/p8/lib/python3.8/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/regen/git_develop/lightdock/lightdock/test/bin/test_dna.py", line 81, in test_lightdock_1diz_30_steps_50_glowworms
    assert filecmp.cmp(
  File "/home/regen/ana/envs/p8/lib/python3.8/filecmp.py", line 52, in cmp
    s2 = _sig(os.stat(f2))
FileNotFoundError: [Errno 2] No such file or directory: '/home/regen/git_develop/lightdock/lightdock/test/bin/scratch_dna_restraints/swarm_0/gso_0.out'

======================================================================
ERROR: lightdock.test.bin.test_dna.TestRegressionDNAShort.test_lightdock_1diz_10_steps_20_glowworms
----------------------------------------------------------------------

Traceback (most recent call last):
  File "/home/regen/ana/envs/p8/lib/python3.8/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/regen/git_develop/lightdock/lightdock/test/bin/test_dna.py", line 41, in test_lightdock_1diz_10_steps_20_glowworms
    assert filecmp.cmp(
  File "/home/regen/ana/envs/p8/lib/python3.8/filecmp.py", line 52, in cmp
    s2 = _sig(os.stat(f2))
FileNotFoundError: [Errno 2] No such file or directory: '/home/regen/git_develop/lightdock/lightdock/test/bin/scratch_dna_short/swarm_0/gso_0.out'

======================================================================
ERROR: lightdock.test.bin.test_pydock.TestRegressionPyDockRestraints.test_lightdock_2uuy_20_steps_25_glowworms_rst
----------------------------------------------------------------------

Traceback (most recent call last):
  File "/home/regen/ana/envs/p8/lib/python3.8/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/regen/git_develop/lightdock/lightdock/test/bin/test_pydock.py", line 40, in test_lightdock_2uuy_20_steps_25_glowworms_rst
    assert filecmp.cmp(
  File "/home/regen/ana/envs/p8/lib/python3.8/filecmp.py", line 52, in cmp
    s2 = _sig(os.stat(f2))
FileNotFoundError: [Errno 2] No such file or directory: '/home/regen/git_develop/lightdock/lightdock/test/bin/scratch_pydock_restraints/swarm_0/gso_0.out'
```

I turned  `lightdock.structure.nm.calculate_nmodes`'s raise statement to a print statement,the different atoms are:

```log
# 1DIZ_rec
# ... above are same
ARG,56,HB2 ,874 - Atom HB2 (index 874)
ARG,56,HB3 ,875 - Atom HB3 (index 875)
THR,58,N   ,893 - Atom N (index 876)
THR,58,CA  ,894 - Atom CA (index 877)
THR,58,C   ,895 - Atom C (index 878)
THR,58,O   ,896 - Atom O (index 879)
THR,58,CB  ,897 - Atom CB (index 880)
----------
THR,58,OG1 ,898 - Atom CG (index 881)
THR,58,CG2 ,899 - Atom ND1 (index 882)
THR,58,HG21,900 - Atom CD2 (index 883)
THR,58,HG22,901 - Atom CE1 (index 884)
THR,58,HG23,902 - Atom NE2 (index 885)
THR,58,H   ,903 - Atom HE1 (index 886)
THR,58,HG1 ,904 - Atom HD1 (index 887)
THR,58,HB  ,905 - Atom H (index 888)
THR,58,HA  ,906 - Atom HD2 (index 889)
LEU,59,N   ,907 - Atom HA (index 890)
LEU,59,CA  ,908 - Atom HB2 (index 891)
LEU,59,C   ,909 - Atom HB3 (index 892)
LEU,59,O   ,910 - Atom N (index 893)
LEU,59,CB  ,911 - Atom CA (index 894)
LEU,59,CG  ,912 - Atom C (index 895)
LEU,59,CD1 ,913 - Atom O (index 896)
LEU,59,CD2 ,914 - Atom CB (index 897)
LEU,59,HD21,915 - Atom OG1 (index 898)
LEU,59,HD22,916 - Atom CG2 (index 899)
LEU,59,HD23,917 - Atom HG21 (index 900)
LEU,59,H   ,918 - Atom HG22 (index 901)
LEU,59,HD11,919 - Atom HG23 (index 902)
LEU,59,HD12,920 - Atom H (index 903)
LEU,59,HD13,921 - Atom HG1 (index 904)
LEU,59,HA  ,922 - Atom HB (index 905)
LEU,59,HG  ,923 - Atom HA (index 906)
LEU,59,HB2 ,924 - Atom N (index 907)
LEU,59,HB3 ,925 - Atom CA (index 908)
ILE,61,N   ,943 - Atom C (index 909)
ILE,61,CA  ,944 - Atom O (index 910)
ILE,61,C   ,945 - Atom CB (index 911)
...

```

If I

1. disable  `lightdock.structure.nm.calculate_nmodes`'s raise statement,or
2. delete the added brackets in `lightdock.structure.complex.Complex.get_atoms`
 
all tests passed,but they don't solve the actual problem either.

As I'm not familar with prody's behavior,I'm not sure what's the cause of this bug.

**Expected behavior**

After adding brackets in  `lightdock.structure.complex.Complex.get_atoms` defination,all tests should pass.

**Screenshots**

![bug](https://user-images.githubusercontent.com/60164864/228580898-73689b75-4854-4222-8590-741028250187.png)

**Desktop (please complete the following information):**
 - OS: manjaro
 - python 3.8

**Comment by brianjimenez** [comment]
Expected to be fixed on version 1.0.0

**Comment by brianjimenez** [comment]
Bug has been fixed in `1.0.0` branch by using `is_protein()` instead of `is_standard()`. Weird that they were not methods, but properties. Thanks for catching this one, @alchemistcai !
